.prettierrc

```diff
@@ -1 +1,3 @@
-{}
+{
+  "singleQuote": true
```

Comment:
so important, especially for me ))

(i had a lot of nerves even from the very first days in Mate, until i found how to disable DOUBLEQUOTES)